pom.xml

```diff
@@ -633,7 +633,6 @@
       <plugin>
         <groupId>com.gitlab.janecekpetr</groupId>
         <artifactId>embedded-postgresql-maven-plugin</artifactId>
-        <version>0.1.0</version>
```

Comment:
I am curious why the version was removed.

Reply:
> I am curious why the version was removed.

Cause we have had a problems with building branch on rancher and by removing the version we've solved the problem and this change doesn't affect anything, so it's okay here. And removing the version the maven automatically uses the last one.

Reply:
I disagree that this is OK. This breaks build reproducibility. I suggest we discover why building branches on rancher is broken rather than making our POM files ambiguous.

Reply:
Ok let's put it back.

Reply:
> I disagree that this is OK. This breaks build reproducibility. I suggest we discover why building branches on rancher is broken rather than making our POM files ambiguous.

I've returned the version.

Reply:
> I disagree that this is OK. This breaks build reproducibility. I suggest we discover why building branches on rancher is broken rather than making our POM files ambiguous.

@mreno-EBSCO could you approve the code review please if it's okay now on your opinion.